bundle-workflow/jenkins_workflow/integtest/Jenkinsfile

```diff
@@ -0,0 +1,72 @@
+pipeline {
+    agent none
+    tools {
+        jdk "JDK14"
+        maven "maven-3.8.2"
+    }
+    stages {
+        stage('Parameters') {
+            steps {
+                script {
+                    properties([
+                            parameters([
+                                    string(
+                                            defaultValue: '',
+                                            name: 's3_bucket',
+                                            trim: true
```

Comment:
This can be read from environment right? Does it need to be a parameter?

Reply:
Yeah, the idea is test orchestrator tells which bucket to run with. And jobs are agnostic of the ENV even though they have access to it. This helps when a user wants to submit a job and use a different S3 bucket.

Reply:
Changing bucket may not work because the jenkins role won't have permissions to read from a different s3 bucket. The role is also configured in jenkins env variables.

Reply:
Thinking about it. This actually makes sense. May be lets not optimize for other use cases let one flow work concretely. I'll use the ENV variable.